docs/user/requirements.txt

```diff
@@ -1,3 +1,4 @@
+black==22.12.0
```

Comment:
Is this actually needed for the site publishing changes?

Reply:
I noticed it is in #362 as well. I think it makes more sense to get in there.

Reply:
Sounds good I'll merge that Pr first and remove it from here.

Black is used for some additional checks within mkdocs when building.

Reply:
Removed